hack/golangci_lint.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+source hack/common.sh
+
+# check for golangci-lint in *.go files
+
+check=0
+
+test_golangci() {
+        "${OUTDIR_TOOLS}"/golangci-lint run ./...
+        if [ $? -ne 0 ]; then
+                check=1
+        fi
+}	
+
+echo "Installing golangci-lint"
+
+wget -O- -nv https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b "$(go env GOPATH)"/openshift/ocs-operator/build/_output/tools v1.31.0
```

Comment:
This is incorrect. You don't want to use GOPATH, you want to use `${OUTDIR_TOOLS}`.